### PR TITLE
mysqld_exporter: update to v0.13.0-rc.0

### PR DESCRIPTION
**Pull request by ymmt2005**
and add `moco-backup` image for backup and restore job of MOCO.

### CODE REVIEW

**Comment by ymmt2005** [comment]
CI log: https://app.circleci.com/pipelines/github/cybozu/neco-containers/1431/workflows/f6d70a15-31ca-49ba-89f5-94a8affcd781/jobs/68703

**Comment by morimoto-cybozu** [comment]
@ymmt2005 
The diff looks good to me.
Would you update `maintenance.md` to add the description of moco-backup and mysqlsh, and to renew the description of mysqld_exporter?
Or do you plan to update it in GA?

**Comment by ymmt2005** [comment]
@morimoto-cybozu Sure, done, PTAL.

File: moco-mysql/moco-backup/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# moco-mysql container
```
Comment (morimoto-cybozu) [inline moco-mysql/moco-backup/Dockerfile:1]:
```suggestion
# moco-backup container
```

File: maintenance.md [hunk 2]
```diff
@@ -428,17 +428,44 @@ moco-mysql is used for [MOCO](https://github.com/cybozu-go/moco).
 The MySQL versions are the ones supported by MOCO. So the versions need not update usually.
 In the regular update, only update the ubuntu base image.
 
-1. Update all `TAG` files in sub directory.
+1. Update all `TAG` files in sub directories.
+
+### moco-backup
+
+`moco-mysql/moco-backup` directory contains a container image used for MOCO's backup/restore feature.
+It should includes `mysql` and `mysqlbinlog` binaries from the latest MySQL and MySQL shell of the
```
Comment (morimoto-cybozu) [inline maintenance.md:436]:
```suggestion
It should include `mysql` and `mysqlbinlog` binaries from the latest MySQL and MySQL shell of the
```

File: maintenance.md [hunk 3]
```diff
@@ -428,17 +428,44 @@ moco-mysql is used for [MOCO](https://github.com/cybozu-go/moco).
...
+same version.
+
+The image tag in `TAG` file should therefore named after the latest MySQL.  For instance,
```
Comment (morimoto-cybozu) [inline maintenance.md:439]:
```suggestion
The image tag in `TAG` file should therefore be named after the latest MySQL.  For instance,
```

File: maintenance.md [hunk 4]
```diff
@@ -428,17 +428,44 @@ moco-mysql is used for [MOCO](https://github.com/cybozu-go/moco).
...
+if the latest MySQL version is 8.0.24, the TAG should be `8.0.24.x` where x is an integer > 1.
```
Comment (morimoto-cybozu) [inline maintenance.md:440]:
```suggestion
if the latest MySQL version is 8.0.24, the TAG should be `8.0.24.x` where x is an integer >= 1.
```

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]
A few hair-splitting comments.

**Comment by ymmt2005** [comment]
@morimoto-cybozu PTAL

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

**Pull request merged**